packages/react/src/auto/polaris/PolarisAutoTable.tsx

```diff
@@ -110,7 +110,18 @@ export const PolarisAutoTable = <
         {rows &&
           columns &&
           rows.map((row, index) => (
-            <IndexTable.Row key={row.id as string} id={row.id as string} position={index}>
+            <IndexTable.Row
+              key={row.id as string}
+              id={row.id as string}
+              position={index}
+              onClick={
+                props.onClickRow
+                  ? () => {
+                      props.onClickRow?.(row);
```

Comment:
this can just be `() => props.onClickRow?.(row)`

Reply:
This will override the default Polaris behaviour even if the `onClickRow` is undefined:
<img width="928" alt="CleanShot 2024-07-19 at 10 03 31@2x" src="https://github.com/user-attachments/assets/cf5a27df-c528-40fc-b76b-a1be9948ded7">
https://polaris.shopify.com/components/tables/index-table#props

Reply:
Makes sense. Didn't realize 👍